cfpb/cfgov-refresh
# Mega menu: heading indentation and class name

Indents headings in mega menu and updates heading/footer class names.

## Additions

- Adds css to indent headings in mega menu
- Updates nemo css to include indentation

## Testing

1. Check out branch
2. Run `gulp styles`
3. Verify that headings have same indentation as nav items in `Consumer Tools` vertical
4. Verify that headings/footer otherwise display as on production at desktop and mobile sizes

## Screenshots

<img width="998" alt="screen shot 2017-08-23 at 7 42 31 am" src="https://user-images.githubusercontent.com/778171/29718411-8feb010e-8967-11e7-8dc3-00d3d980d5a5.png">


## Checklist
